docker/pypi/wmagent-mariadb/mariadb-docker-run.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+### This script is to be used for running the Mariadb docker container at a VM
+### Its sole purpose is to set all the needed mount points from the Host VM and
+### forward all Mariadb runtime parameters to the Mariadb container entrypoint run.sh
+### It accepts only the set of parameters relevant to Mariadb's container run.sh
+### and no build dependent ones. The docker image tag to be searched for execution is
+### always `latest`.
+
+
+# NOTE: In the help call to the current scrit we only repeat the help and usage
+#       information for all the parameters accepted by run.sh.
+help(){
+    echo -e $*
+    cat <<EOF
+
+The script to be used for running a Mariadb docker container at a VM. The full set of arguments
+passed to the current script are to be forwarded to the Mariadb container entrypoint 'run.sh'
+
+Usage: mariadb-docker-run.sh [-t <team_name>] [-n <agent_number>] [-f <db_flavour>]
```

Comment:
Can you update the usage example, to maybe this?
```suggestion
Usage: mariadb-docker-run.sh [-p] [-t <mariadb_tag>]
```